.travis.yml

```diff
@@ -16,9 +16,10 @@ before_install:
   # Useful for debugging any issues with conda
   - conda info -a
 install:
-  # Create test environment and install deps
-  - conda env create -n test-environment python=$TRAVIS_PYTHON_VERSION -f environment.yml
-  - source activate test-environment
+  # Create test environment and install deps, prefer defaults, use bioconda when needed
+  - conda create -n testenv -c defaults -c bioconda python=$TRAVIS_PYTHON_VERSION numpy cython h5py
```

Comment:
```suggestion
  - conda create -n testenv python=$TRAVIS_PYTHON_VERSION numpy cython h5py
```

Reply:
understood... there is nothing `bioconda`-related among `numpy,cython,h5py` ...